pkg/k8s/watchers/cilium_endpoint_slice.go

```diff
@@ -58,7 +81,13 @@ func (k *K8sWatcher) ciliumEndpointSliceInit(client *k8s.K8sCiliumClient, asyncC
 				},
 			},
 			nil,
+			cache.Indexers{
+				"localNode": CreateCiliumEndpointSliceLocalPodIndexFunc(),
+			},
 		)
+		k.ciliumEndpointSliceStoreMU.Lock()
+		k.ciliumEndpointSliceStore = cesIndexer
```

Comment:
Might worth renaming this `ciliumEndpointSliceStore` to `ciliumEndpointSliceIndexer`?